### PR TITLE
1.20.1 Change the tooltip of the ME Pattern Provider

### PR DESCRIPTION
**Pull request by xingluo01**
## What
_Change the tooltip of the ME Pattern Provider_

## Outcome
_Change the tooltip of the ME Pattern Provider_

### CODE REVIEW

File: src/main/java/com/gregtechceu/gtceu/data/lang/MachineLang.java [hunk 1]
```diff
@@ -724,7 +724,7 @@ protected static void init(RegistrateLangProvider provider) {
         provider.add("gtceu.machine.fluid_hatch.import.tooltip", "Fluid Input for Multiblocks");
         provider.add("gtceu.machine.fluid_hatch.export.tooltip", "Fluid Output for Multiblocks");
         provider.add("block.gtceu.pattern_buffer.desc.0",
-                "§fAllows direct §6AE2 pattern storage §ffor GregTech Multiblocks.");
+                "§fProvides better §6Pattern Provider §ffor Multiblocks.");
```
Comment (Ghostipedia) [inline src/main/java/com/gregtechceu/gtceu/data/lang/MachineLang.java:727]: Do not change this

File: src/main/java/com/gregtechceu/gtceu/data/lang/MachineLang.java [hunk 2]
```diff
@@ -740,11 +740,9 @@ protected static void init(RegistrateLangProvider provider) {
 
         provider.add("gui.gtceu.share_inventory.title", "Shared Item Inventory");
         provider.add("gui.gtceu.share_inventory.desc.0", "Shares inserted items with all patterns within buffer!");
-        provider.add("gui.gtcey.share_inventory.desc.1", "Allows powerful automation by storing catalysts");
-        provider.add("gui.gtceu.share_tank.title", "Shared Tank Inventory");
-        provider.add("gui.gtceu.share_tank.desc.0",
-                "Shares inserted fluids/gasses/etc. with all patterns within buffer!");
         provider.add("gui.gtceu.share_inventory.desc.1", "Allows powerful automation by storing catalysts");
+        provider.add("gui.gtceu.share_tank.title", "Shared Tank Inventory");
```
Comment (Ghostipedia) [inline src/main/java/com/gregtechceu/gtceu/data/lang/MachineLang.java:744]: or this

File: src/main/java/com/gregtechceu/gtceu/integration/ae2/machine/MEPatternBufferPartMachine.java [hunk 3]
```diff
@@ -264,7 +264,7 @@ public void attachConfigurators(ConfiguratorPanel configuratorPanel) {
                 shareInventory.storage, Component.translatable("gui.gtceu.share_inventory.title"))
                 .setTooltips(List.of(
                         Component.translatable("gui.gtceu.share_inventory.desc.0"),
-                        Component.translatable("gui.gtcey.share_inventory.desc.1"))));
+                        Component.translatable("gui.gtceu.share_inventory.desc.1"))));
```
Comment (Ghostipedia) [inline src/main/java/com/gregtechceu/gtceu/integration/ae2/machine/MEPatternBufferPartMachine.java:267]: ty for catching this

**Review by Ghostipedia — CHANGES_REQUESTED** [review]
Do not change any english lang beyond the broken gtcey -> gtceu component

**Comment by xingluo01** [comment]
oh

**Pull request closed without merging**